osmdroid-android/src/main/java/org/osmdroid/tileprovider/MapTileProviderBasic.java

```diff
@@ -78,6 +78,14 @@ public MapTileProviderBasic(final IRegisterReceiver pRegisterReceiver,
 			final INetworkAvailablityCheck aNetworkAvailablityCheck, final ITileSource pTileSource,
 			final Context pContext, final IFilesystemCache cacheWriter) {
 		super(pTileSource, pRegisterReceiver);
+		init(pRegisterReceiver, aNetworkAvailablityCheck, pTileSource, pContext, cacheWriter);
+	}
+
+	protected void init(IRegisterReceiver pRegisterReceiver,
```

Comment:
I can't see the point of new method `init`. I thought the purpose was mainly to override the new `create*` methods.